anonymized_text.txt

```diff
@@ -0,0 +1,3 @@
+Hello my name is <ANONYMIZED> , my phone number is ***********. Let me share with you my email 
```

Comment:
remark: this is not your real physical address, right? if it is, please change it to avoid sharing personal details on github

Reply:
nope of course it's not xd